### PR TITLE
Improve wording, provide reference to B'day attack.

### PR DESCRIPTION
**Pull request by donatello**

### CODE REVIEW

File: DARE-1.1.md [hunk 1]
```diff
@@ -257,29 +257,31 @@ generator (CRSPRNG) should be used to generate `R`. Most operating systems
...
 
 ### 5.2 Side channels
 
 The decrypted payload of a package should never be processed before the
-authentication  tag of the package is not verified. Any decrypted but not
-yet verified data could be modified by an adversary. Additionally the
-authentication tag should be verified using a constant-time comparison
-to prevent a timing side channel. This is ideally done by the AEAD cipher
+authentication tag of the package is verified. Any such decrypted but as yet
+un-verified data could have be modified by an adversary. Additionally, the
```
Comment (kannappanr) [inline DARE-1.1.md]: `could have been`

File: DARE-1.1.md [hunk 2]
```diff
@@ -257,29 +257,31 @@ generator (CRSPRNG) should be used to generate `R`. Most operating systems
...
+authentication tag should be verified using a constant-time comparison to
+prevent a timing side channel. This is ideally done by the AEAD cipher
```
Comment (kannappanr) [inline DARE-1.1.md]: `prevent a timing side channel attack`?

File: DARE-1.1.md [hunk 3]
```diff
@@ -257,29 +257,31 @@ generator (CRSPRNG) should be used to generate `R`. Most operating systems
...
 implementation itself.
 
 ## 6. Secret key
 
 The secret key should be unique per encrypted data stream. Since it is 256 bits
-long the probability of choosing a specific key more than once is negligible
-(<code>< ~2<sup>-128</sup></code>) if the secret key is chosen uniformly at
-random. There are multiple ways to choose a secret key:
+long the probability of choosing a previously chosen key again is negligible
```
Comment (kannappanr) [inline DARE-1.1.md]: `Since it is 256 bits long, the probability of choosing a previously used key again is negligible.`

File: DARE-1.1.md (hunk 2 above)
Comment (donatello) [reply DARE-1.1.md]: I think it is not strictly necessary.

**Comment by donatello** [comment]
Updated PR after review.

**Comment by harshavardhana** [comment]
ping @aead are you waiting on something here?

**Comment by aead** [comment]
@harshavardhana This is out of sync - There is a complete new specification for DARE 2.0. (I once published a draft on slack).

**Comment by donatello** [comment]
> @harshavardhana This is out of sync - There is a complete new specification for DARE 2.0. (I once published a draft on slack).

Let's close this then.

**Pull request closed without merging**